source/fundamentals/typescript.txt

```diff
@@ -67,10 +69,13 @@ You can pass a type parameter that extends the ``Document`` interface like this:
       :start-after: start-no-key
       :end-before: end-no-key
 
-Any Type
-~~~~~~~~
+To view an example of a mutually recursive type, which is not supported by the
+preceding classes, see the :ref:`<node-driver-limitations-mutual-recursion>` section.
```

Comment:
Since the classes aren't directly before this text, perhaps it's worth mentioning them again or linking back to it, which is how the style guide suggests using the term "preceding".